StitchCore-iOS/Extensions/PromiseKit/Stitch-PromiseKit/Stitch-PromiseKit/Adapters.swift

```diff
@@ -0,0 +1,37 @@
+import PromiseKit
+
+// swiftlint:disable identifier_name
```

Comment:
[nit] we could just rename `t` and `e` to something like `value` and `error` to be more readable in general